### DISCUSSION TITLE
Laragon network security on Windows 10

### DISCUSSION THREAD

**Discussion by scotmalone** [leokhoa/laragon#541] [Q&A]
Laragon has built in security yes? On Windows, Windows Defender asked me what networks to allow Laragon on. It defaulted to checking both public and private so I clicked OK and Windows did not ask me again. Is this OK or should I be concerned? How can I changed the setting if needed? Laragon is useful and great. Thank you.

**Comment by Pen-y-Fan** [comment]
Its part of the setup process, see my blog: [How to set up Laragon on a new Windows computer (part 2) - permissions](https://pen-y-fan.github.io/2022/11/08/how-to-set-up-laragon-on-a-new-windows-computer-part-2/).. There is no need for concern.

**Comment by scotmalone** [comment]
Thank you. Where/how can I learn more of the details about Laragon with respect to security on Windows?

**Reply by Pen-y-Fan** [reply]
I'm not 100% sure of the question, I think you asking about Windows firewall security. Or how to check which ports are open and in use.

- [Best practices for configuring Windows Defender Firewall](https://learn.microsoft.com/en-us/windows/security/operating-system-security/network-security/windows-firewall/best-practices-configuring)
- To check which ports are in use run **netstat** (Laragon menu > tools > netstat) and wait for Notepad++ to open with the results

Laragon is recommended for local app development, it is safe to use on a Windows PC, but it isn't recommended for a Windows server, although you are not asking about hosting security, I guess you are not asking about that.

Please advise more details if you mean something else, and I'll try and help.
